modules/phase_field/src/utils/PolycrystalICTools.C

```diff
@@ -251,28 +256,26 @@ PolycrystalICTools::buildElementalGrainAdjacencyGraph(
 
       if (!set_intersection.empty())
       {
-        adjacency_matrix[i][j] = true;
-        adjacency_matrix[j][i] = true;
+        adjacency_matrix(i, j) = true;
```

Comment:
Well, I really wanted to keep PETSc specific stuff out of MOOSE. I probably should assign Real values though at least.